Dockerfile

```diff
@@ -6,7 +6,7 @@
 FROM node:12.18.3-buster-slim@sha256:dd6aa3ed10af4374b88f8a6624aeee7522772bb08e8dd5e917ff729d1d3c3a4f
     
 RUN  apt-get update \
-     && apt-get install -y wget gnupg ca-certificates \
+     && apt-get install -y wget gnupg ca-certificates libxss1 \
```

Comment:
I think this is still relevant even with Node.js `14.x`, but you might want to rebase (now that version `14.x` made it into `master`) and check again.

Reply:
Yeah our builds have been failing since the update to 14.x 

![image](https://user-images.githubusercontent.com/79684/108387975-ff8da880-71c2-11eb-9944-e1a0f56d1d48.png)

Reply:
Then I think you might want to merge this one 😅